nym-vpn-x/.pkg/aur/PKGBUILD

```diff
@@ -0,0 +1,54 @@
+# Maintainer: Pierre Dommerc <pierre@nymtech.net>
+
+pkgname=nymvpn-x
+pkgver=0.1.1
+pkgrel=1
+pkgdesc='NymVPN next desktop client'
+arch=('x86_64')
+url='https://github.com/nymtech/nym-vpn-client'
+license=('GPL-3.0-only')
+depends=('glibc' 'gcc-libs' 'bash' 'cairo' 'pango' 'gtk3' 'webkit2gtk' 'gdk-pixbuf2' 'glib2' 'openssl' 'libsoup')
```

Comment:
Yes, but I think we want it declared as an optional dependency since `depends` array is for dependencies that must be installed for the software to build and **run**. The client app is perfectly able to run without any installed and running daemon.
Also in the future we'll probably ship the daemon to support non-systemd environment, as a docker container for example, so the user should be able to use the client with any daemon implementation he wants.